Sorting/MergeSort.php

```diff
@@ -8,49 +8,45 @@
  */
 function mergeSort(array $arr)
 {
-    if (count($arr) <= 1)
-    {
+    if (count($arr) <= 1) {
```

Comment:
We try to follow the PER style guide, and this is the correct way to put braces for if statements.

https://www.php-fig.org/per/coding-style/#51-if-elseif-else